### ISSUE TITLE
[SF] [Toast] [A11y] Toast message is not announced by JAWS 2020

### ISSUE THREAD

**Issue by i323808** [SAP/ui5-webcomponents#4127]
### **Bug Description**
Toast message is not announced by JAWS 2020.

### **Expected Behavior**
Toast message should be announced by JAWS 2020

### **Steps to Reproduce**
Open JAWS 2020
Open https://sap.github.io/ui5-webcomponents-react/?path=/docs/modals-popovers-toast--default-story
Click 'Show Toast' button
Verify whether the toast message is announced by JAWS 2020

### **Context**
- UI5 Web Components version: 0.31.19
- OS/Platform: macOS
- Browser: Chrome
- Affected component: Toast

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

### **Stakeholder Info** (if applicable)

- Organization: SuccessFactors
